doc/source/user_guide/categorical.rst

```diff
@@ -439,15 +439,19 @@ Sorting and order
 .. _categorical.sort:
 
 If categorical data is ordered (``s.cat.ordered == True``), then the order of the categories has a
-meaning and certain operations are possible. If the categorical is unordered, ``.min()/.max()`` will raise a ``TypeError``.
+meaning and certain operations are possible. If the categorical is unordered, the data can still be sorted,
+but ``.min()/.max()`` will raise a ``TypeError``.
 
 .. ipython:: python
 
     s = pd.Series(pd.Categorical(["a", "b", "c", "a"], ordered=False))
-    s.sort_values(inplace=True)
-    s = pd.Series(["a", "b", "c", "a"]).astype(CategoricalDtype(ordered=True))
-    s.sort_values(inplace=True)
-    s
+    s.sort_values(ascending=False)
+    try:
+        s.min()
+    except TypeError as e:
```

Comment:
It would better to demonstrate this with the `:okexcept:` directive

Reply:
Hm, if I look at examples where the `:okexcept:` directive is used, for instance [here](https://pandas.pydata.org/docs/dev/user_guide/gotchas.html#using-if-truth-statements-with-pandas) or [here](https://pandas.pydata.org/docs/dev/user_guide/advanced.html#renaming-names-of-an-index-or-multiindex), I'm not sure if this is so good an idea in the given case, as it inserts quite a lengthy (~15 lines) traceback which, in my opinion, distracts from reading and makes the example less clear than with `try` / `except`.

Reply:
Could you show an image of how the docs would render if `:okexcept:` was used?

Reply:
see the links I provided in the reply:
```
.. ipython:: python
    :okexcept:

    if pd.Series([False, True, False]):
        print("I was true")
```
renders as 
![grafik](https://user-images.githubusercontent.com/19879328/180955288-b60e6b5c-9687-4197-a6e5-94f5486710bc.png)

and
```
.. ipython:: python
   :okexcept:

   mi.levels[0].name = "name via level"
```
renders as
![grafik](https://user-images.githubusercontent.com/19879328/180955412-2c29d20f-94f6-4407-acbb-59256563f7dc.png)

Reply:
Sorry to clarify, how would the docs you modified in this PR render with `:okexcept:`?

Reply:
```
.. ipython:: python
    :okexcept:

    s = pd.Series(pd.Categorical(["a", "b", "c", "a"], ordered=False))
    s.sort_values(ascending=False)
    s.min()
```
results in
![out](https://user-images.githubusercontent.com/19879328/183150140-73b440d6-f7b5-4b7b-9787-e3341a652c5f.png)

Reply:
I would still be partial to adding `:okexcept:` here. Though verbose, it shows the exact behavior that the user should see.

Reply:
I'm afraid I can't agree here: IMO the purpose here is just to demonstrate that an exception will be raised when trying to do `min` or `max` on a not ordered categorical series. If we then catch it or let it be thrown doesn't play a role for this purpose, so I'd prefer the visually less distracting option of catching it. This is the way it's done all the way in [the given section of the user guide](https://pandas.pydata.org/docs/dev/user_guide/categorical.html) (9 times), why should we treat this 10th time differently? Maybe we can leave the `try/except` here just for uniformity's sake.

Reply:
Personally, I think those other sections should also use `:okexcept:` as well.